deploy/kustomize/kubernetes-distributed-lvm/kustomization.yaml

```diff
@@ -0,0 +1,65 @@
+bases:
+- ../kubernetes-base-lvm
+
+resources:
+- webhooks-rbac.yaml
+
+patchesJSON6902:
+- target:
+    group: apps
+    version: v1
+    kind: StatefulSet
+    name: pmem-csi-controller
+  patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/command/-
+      value: -mode=webhooks
```

Comment:
What does this "webhooks" mode does? Isn't it that valid mods are 'controller' and 'node'?

Reply:
Okay, after going through the design documentation changes, I understood that the new 'webhooks' mode replaces the existing 'controller' mode. But the driver command-line flags in [`pkg/pmem-csi-dirver/main.go`](https://github.com/pohly/pmem-CSI/blob/distributed-provisioning/pkg/pmem-csi-driver/main.go#L38) is not synced with this.

Reply:
Fixed.